README.md

```diff
@@ -39,6 +39,15 @@ Getting started
 
 You obviously want to bind to 0.0.0.0 and port 80 when running in <i>production</i>.
 
+## Testing
+
+For testing speedily along with full coverage report, we recommend running `pytest` on tests folder:
```

Comment:
```suggestion
In order to run the tests and receive a test coverage report, we recommend running `pytest`:
```

Reply:
Thanks for the review, will do the changes shortly :)